defra_ruby_email.gemspec

```diff
@@ -15,30 +15,15 @@ Gem::Specification.new do |s|
   s.summary     = "Defra Ruby on Rails access last email engine"
   s.description = "A Rails engine which can be used to access details of the last email when loaded into an application"
   s.license     = "The Open Government Licence (OGL) Version 3"
-
+  s.required_ruby_version = ">= 3.2"
```

Comment:
Wouldn't this cause an issue for projects that still use earlier ruby versions? As far as I remember most still use version 3.1.2